sphinx_source/Ucuncu_taraf_moduller/kivy_dersleri/events.rst

```diff
@@ -0,0 +1,119 @@
+
+Şimdi bazı belli başlı olayları inceleyelim.
+
+Clock Olayları
+==============
+
+Clock ile ilgili daha önce bir örnek yapmıştık. Clock sınıfı, belirtilen süre sonunda bir fonksiyon çalıştırmak üzere tasarlanmıştır. İşte bu süre sonunda fonksiyonu çalıştırması, bir olay olarak ele alınıyor. Clock için zamanlayıcı da diyebiliriz.
+
+Clock sınıfını iki şekilde kullanabiliriz. Birincisi, fonksiyonun sadece bir kez çağrılması. Yani belirtilen süre sonunda, fonksiyon çalıştırılır ve işlem tamamlanır. Bir örnekle görelim
+
+.. code-block:: python
+	
+	# coding: utf-8
+
+	from kivy.app import App
+	from kivy.clock import Clock
+	from kivy.uix.label import Label
+
+	class Program(App):
+	    def build(self):
+
+	        self.yazi = Label(text = "Merhaba")
+
+	        Clock.schedule_once(self.degistir,3) # 3 saniye sonra, self.degistir adlı fonksiyonu çalıştır
+
+	        return self.yazi
+
+	    def degistir(self,event):
+	        self.yazi.text = "Dünya"
+
+
+	Program().run()
+
+Program başladıktan 3 saniye sonra, ekrandaki "Merhaba" yazısı, "Dünya" yazısıyla değiştirilecektir. Böylece işlem tamamlanmış olacaktır. 
+
+Clock sınıfının bir başka kullanımı ise, belli aralıklarla sürekli çalışmasıdır. Fonksiyonu çalıştırır, sonra belirtilen süre kadar bekler ve sonra tekrar çalıştırır. Bunu, siz sonlandırana kadar veya program kapanana kadar yapmaya devam eder. Saat örneği, bu kullanım için çok iyi bir örnektir.
+
+.. code-block:: python
+
+	# coding: utf-8
+
+	from kivy.app import App
+	from kivy.clock import Clock
+	from kivy.uix.label import Label
+	import datetime
+
+	class Program(App):
+	    def build(self):
+	        self.yazi = Label(text = "Merhaba",markup = True)
+
+	        Clock.schedule_interval(self.degistir,1) # 3 saniye sonra, self.degistir adlı fonksiyonu çalıştır
+
+	        return self.yazi
+
+	    def degistir(self,event):
+	        zaman = datetime.datetime.now()
+	        self.saat = "[size=25sp]{}:{}:{}[/size]".format(zaman.hour,zaman.minute,zaman.second)
+	        self.yazi.text = self.saat
+
+
+	Program().run()
+
+Programda kullandığımız datetime modülü, zaman bilgisini elde etmek içindir. Dikkat ettiyseniz, sürekli çalışmasını istediğimizde Clock.schedule_interval() kullanıyoruz. Bir saniye aralıkla çalışmasını sağlayarak, ekranda zaman bilgisini göstermiş olduk. 
+
+Her iki kullanım şeklinde dikkat etmemiz gereken şey, çalıştırılacak olan fonksiyonun mutlaka bir argüman almasıdır. Burda kullandığımız "self" dışında alması gereken bir argüman olduğunu görüyoruz. Bu argüman, belirlediğimiz zamanın kendisidir.
+
+Eğer tanımladığınız bir Clock nesnesini kaldırmak isterseniz;
+
+.. code-block:: python
+
+	# Tanımladık
+	self.event = Clock.schedule_interval(self.fonk,2)
+	self.event2 = Clock.schedule_once(self.fonk2,5)
+
+	# Ve bunları kaldırıyoruz
+	self.event.cancel()
+	Clock.unschedule(self.event2)
+
+Input Events
+==================
```

Comment:
Başlık altı karakterler fazla. Diğer başlık altı karakterlerden sorun olanları da düzeltebilir misiniz?

Reply:
Aslında çok önemli değil ancak inşa sırasında uyarı veriyor. Bu arada belgelerinizi, göndermeden önce inşa (build) ettiniz mi?